lib/flutter_map_tappable_polyline.dart

```diff
@@ -63,14 +63,14 @@ class TaggedPolyline extends Polyline {
       isDotted = false,
       this.tag})
       : super(
-            points: points,
-            strokeWidth: strokeWidth,
-            color: color,
-            borderStrokeWidth: borderStrokeWidth,
-            borderColor: borderColor,
-            gradientColors: gradientColors,
-            colorsStop: colorsStop,
-            isDotted: isDotted);
+      points: points,
+      strokeWidth: strokeWidth,
+      color: color,
+      borderStrokeWidth: borderStrokeWidth,
+      borderColor: borderColor,
+      gradientColors: gradientColors,
+      colorsStop: colorsStop,
+      isDotted: isDotted);
```

Comment:
Could you revert this. It's about code readability

Reply:
I cannot get dartfmt to format it that way. Using dartfmt inside android studio..

Reply:
Also, as the package analyzer will check for formatting issues i don't see the issue?

Reply:
Push a commit myself to see

Reply:
The action was misconfigured. It did not fail if the score was too low.